### PR TITLE
lefthandtools-marginfix, Fix for issue #587

### PR DESCRIPTION
**Pull request by mulfvik**
#587

### CODE REVIEW

**Comment by mulfvik** [comment]
@daRasmussen @jokd @tonnyandersson Any thoughts on this ?

File: scss/ui/_flex.scss [hunk 1]
```diff
@@ -69,4 +69,4 @@ $flex-grow-all: 2;
   .align-stretch {
     align-items: stretch;
   }     
-}
\ No newline at end of file
+}
```
Comment (daRasmussen) [inline scss/ui/_flex.scss]: is this change necessary?

Comment (mulfvik) [reply scss/ui/_flex.scss]: @daRasmussen I guess it was a new line at the end issue together with margin change it shows that row 72 was change but it was actually a new row added on row 73 at first commit. 
So to answer your question: yes and no `:)`
